src/cleaners/lead-image-url.test.js

```diff
@@ -9,8 +9,7 @@ describe('clean(leadImageUrl)', () => {
   });
 
   it('returns null if the url is not valid', () => {
-    const url = 'this is not a valid url';
```

Comment:
In addition to inlining fixtures, I went a bit further in many files and tended to inline short strings that were only used on the line immediately following them. I think this change is in the spirit of the major focus of this PR.